grpc/src/main/scala/io/github/positionpal/grpc/GrpcServer.scala

```diff
@@ -20,17 +26,21 @@ class GrpcServer(using system: ActorSystem[?]):
       ServerReflection.partial(List(ChatService)),
     )
 
-  def run(): Future[ServerBinding] =
-    val bound = Http().newServerAt(interface = "127.0.0.1", port = 8080).bind(service)
+  def startup(): Future[ServerBinding] =
+    val config = ConfigFactory.load()
+    val port = config.getInt("grpc.service.default-grpc-port")
+    val interface = config.getString("grpc.service.grpc-bind-interface")
+
+    val bound = Http().newServerAt(interface = interface, port = port).bind(service)
       .map(_.addToCoordinatedShutdown(hardTerminationDeadline = 10.seconds))
 
     import scala.util.{Failure, Success}
     bound.onComplete:
       case Success(binding) =>
         val address = binding.localAddress
-        println(s"gRPC server bound to ${address.getHostString}:${address.getPort}")
+        logger.info(s"gRPC server bound to ${address.getHostString}:${address.getPort}")
       case Failure(ex) =>
-        println("Failed to bind gRPC endpoint, terminating system")
+        logger.info("Failed to bind gRPC endpoint, terminating system")
```

Comment:
Error more appropriate?
```suggestion
        logger.error("Failed to bind gRPC endpoint, terminating system")
```

Reply:
yhea, sorry, you're right